index.html

```diff
@@ -2,7 +2,108 @@
 <html>
   <head>
     <title>Mashup</title>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
+    <style>
+	    input {
+	    	margin: 10px; 
+	    }
+
+	    li{
+	    	list-style: none; 
+	    }
+
+	    .getimages {
+	    	width: 100px;
+	    	border : 1px solid black ;
+	    	cursor: pointer;
+	    	margin: 5px; 
+	    }
+
+	    .queriedImages {
+	    }
+
+    </style>
   </head>
+
   <body>
+ 	<input type = "text" class = "inputText"></input>
+ 	<button type = "submit"> Get images</button>
+ 	<button type = "reset"> Reset images</button>
+ 	<div class= "widget"></div>
+ 	<div class= "queriedImages"></div>
+
+
+  <script>
+	function loadImages(queryTag) {
+		$.ajax({
+		url: 'https://api.instagram.com/v1/tags/'+ queryTag +'/media/recent?client_id=ce95cb4e56c146c994457b48a839f6a8',
+		dataType: 'jsonp',
+		success: function(result) {
+		  for (var i = 0; i < result.data.length; i++) {
+		    var url = result.data[i].images.thumbnail.url;
+		    $('.queriedImages').append('<img class =\'EachImage\' src="' + url + '"/>');
+		  }
+		}
+		});
+	}
+
+	function searchArtist(query) {
+		$.ajax({
+			url:"https://api.spotify.com/v1/search?q="+ query +"&type=artist",
```

Comment:
Ditto. For query parameters, also take a look at `$.ajax()`'s `data` setting.

http://api.jquery.com/jquery.ajax/